web/api/report_server.thrift

```diff
@@ -852,6 +863,14 @@ service codeCheckerDBAccess {
                                  5: i64          offset)
                                  throws (1: codechecker_api_shared.RequestFailed requestError),
 
+  // It gives statistics for specific runs which checkers were
+  // enabled and how many reports are opened or closed.
+  // If the run id list is empty the statistics
+  // will be counted for all of the runs.
+  CheckerInfos getCheckerInfo(1: list<i64>    runIds,
```

Comment:
Also, this function (and the associated return struct) could've really used a much better name... `CheckerInfo` is as vague as it can get, especially since this API function seems to conflate multiple things into the same return object (both details about a checker, which could've been its own function; and the number of bugs specific to the checkers, which also could've been its own function).

Maybe `getCheckerStatusVerificationDetails()`... At least that will immediately make it apparent that this is a **very** purpose-tailored function.